gen_cert.py

```diff
@@ -1274,6 +1125,22 @@ def _generate_v4_certificate(
             achievements_string = grade_interstitial.decode('utf-8').format(grade=grade_html) + '<br /><br />'
         achievements_paragraph = u"{0}{1}".format(achievements_string, achievements_description_string)
 
+        # Overide achievements/interstitial strings with yaml designations info based on designation
+        designation_tag = ''
+        if designation:
+            # Add student name designation if it exists
+            student_name = u"{name}, {designation}".format(
+                name=student_name,
+                designation=designation.decode('utf-8'),
+            )
+            if 'designations' in self.cert_data:
+                for key, value in self.cert_data.get('designations').iteritems():
+                    if designation in value['titles']:
+                        achievements_string = value['credits']
+                        achievements_description_string = self.cert_data.get('CREDITS')
```

Comment:
Yes, if a cert has 'titles' then it has associated 'credits' and 'CREDITS'.  And, sure, courses may want titles and not credits but this seems like a new feature without a current use.   

Related info,  'CREDITS' is not in the cme_default template.  As far as I know we can't make a pure 'cme_default' cert and all cme_default derivatives have 'CREDITS', but it seems odd that our default is missing a required element.

Let me know, I can put up a new commit with this fix and/or feature.